wiki/Tournaments/oMMFF/1/en.md

```diff
@@ -1145,18 +1222,18 @@ This competition has come to an end and resulted in the following podium:
   - **DoubleTime** - The DoubleTime mod will be used by both players if the chosen beatmap is in the DoubleTime category of the mappool. They may also add HD if they wish to.
   - **FreeMod** - Each player may use the mod they want if the chosen beatmap is in the Tiebreaker category of the mappool.
 
-#### Mod Usage for osu!mania
+#### Mod usage for osu!mania
 
 - For osu!mania, the tournament will use the **FreeMod** rule on all picks. This allows **Hidden**, **Fade In** or **Flashlight** to be used during a pick.
 
-### Eligibility and Registration
+### Eligibility and registration
 
-- Any player who has a flag of a country that is part of the [Francophonie](http://www.world-territories.com/ttfr/dossiers.php?dossier=francophonie) or part of the [Overseas France](https://fr.wikipedia.org/wiki/France_d%27outre-mer) is allowed to play in this tournament.
+- Any player who has a flag of a country that is part of the [Francophonie](http://www.world-territories.com/ttfr/dossiers.php?dossier=francophonie "World Territories") or part of the [Overseas France](https://fr.wikipedia.org/wiki/France_d%27outre-mer "Wikipedia") is allowed to play in this tournament.
```

Comment:
Their [terms of use](http://www.world-territories.com/ttfr/cutilisation.php):

![image](https://user-images.githubusercontent.com/36758269/124268015-45775780-db39-11eb-83fe-b3437a7200a6.png)


```suggestion
- Any player who has a flag of a country that is part of the [Francophonie](http://www.world-territories.com/ttfr/dossiers.php?dossier=francophonie "Tous les territoires français") or part of the [Overseas France](https://fr.wikipedia.org/wiki/France_d%27outre-mer "Wikipedia") is allowed to play in this tournament.
```